Error occur when cmake kernels_avx512.cc.o

In file included from /workspace/CTranslate2/build/kernels_avx512.cc:7:0:
/workspace/CTranslate2/src/cpu/vec_avx512.h: In static member function 'static ctranslate2::cpu::Vec<float, (ctranslate2::cpu::CpuIsa)3>::mask_type ctranslate2::cpu::Vec<float, (ctranslate2::cpu::CpuIsa)3>::lt(ctranslate2::cpu::Vec<float, (ctranslate2::cpu::CpuIsa)3>::value_type, ctranslate2::cpu::Vec<float, (ctranslate2::cpu::CpuIsa)3>::value_type)':
/workspace/CTranslate2/src/cpu/vec_avx512.h:55:16: error: '_mm512_cmplt_ps_mask' was not declared in this scope
         return _mm512_cmplt_ps_mask(a, b);
                ^~~~~~~~~~~~~~~~~~~~
/workspace/CTranslate2/src/cpu/vec_avx512.h:55:16: note: suggested alternative: '_mm512_cmp_ps_mask'
         return _mm512_cmplt_ps_mask(a, b);
                ^~~~~~~~~~~~~~~~~~~~
                _mm512_cmp_ps_mask
CMakeFiles/ctranslate2.dir/build.make:1419: recipe for target 'CMakeFiles/ctranslate2.dir/kernels_avx512.cc.o' failed

--- a/src/cpu/vec_avx512.h
+++ b/src/cpu/vec_avx512.h
@@ -64,7 +64,7 @@ namespace ctranslate2 {
       }
 
       static inline mask_type lt(value_type a, value_type b) {
-        return _mm512_cmplt_ps_mask(a, b);
+        return _mm512_cmp_ps_mask(a, b, _CMP_LT_OS);
       }
 
       static inline value_type select(mask_type mask, value_type a, value_type b) {

